### ISSUE TITLE
[Feature Request] An option to run an external command with the IMDb ID and filename

### ISSUE THREAD

**Issue by chocolateboy** [BurntSushi/imdb-rename#15]
**TL; DR**: any chance of adding support for running an external command after each rename? This would provide an easy way to hook into imdb-rename's rich metadata without needing to navigate its underlying database, e.g.:

    $ imdb-rename --exec ./tag.sh ...

could call:

    $ ./tag.sh "/media/movies/big-buck-bunny-720p.mp4" "tt1254207" "renamed"

or:

    $ ./tag.sh "/media/movies/Big Buck Bunny (2008).mp4" "tt1254207" "skipped"

---

When watching TV episodes (and occasionally when watching movies), I often find myself wanting to look up some info such as the writer(s), and end up pausing the video and laboriously clicking through several pages on IMDb/Wikipedia to find out.

I've been thinking of writing a script to go through my files adding the episode/movie's IMDb ID — effectively its GUID — to each one so I can look up or attach this information automatically, but that seems redundant given I've already got the IDs stored on my system in imdb-rename's database, and I've most likely already used it to rename each file.

It'd be great if there was a way to intercept this morsel of metadata while imdb-rename is running. I know everyone's needs would be different, so I'm not suggesting adding a specific command for each use case, but rather a single command which is passed:

- the path to the file
- its IMDb ID
- the action/status, e.g. "rename"/"renamed" or "skip"/"skipped"

Ideally, I'd like to be able to run the command even if the filename is correct/unchanged (to tag files I've already renamed), hence the third argument to distinguish those cases.
